target/linux/ath79/image/generic.mk

```diff
@@ -1283,6 +1283,16 @@ define Device/sitecom_wlr-8100
 endef
 TARGET_DEVICES += sitecom_wlr-8100
 
+define Device/telco_t1
+  SOC := qca9531
+  DEVICE_VENDOR := Telco
+  DEVICE_MODEL := T1
+  DEVICE_PACKAGES := kmod-leds-gpio kmod-usb2 kmod-usb-net kmod-usb-net-qmi-wwan \
+	kmod-usb-serial kmod-usb-serial-option uqmi -swconfig -uboot-envtools
```

Comment:
kmod-usb-net and kmod-usb-serial can be dropped here, as they will be pulled by the other packages.